app/styles/modules/_home.scss

```diff
@@ -0,0 +1,378 @@
+    margin: 0 auto $double-gap auto;
+    width: $default-grid * 3;
+  }
+
+  .arrow-wrapper {
+    bottom: $double-gap;
+    height: $default-grid;
+    position: absolute;
+    width: 100%;
+
+    .arrow {
+      @include hidpi-background-image(arrow, 44px 18px);
+      background-position: center center;
+      background-repeat: no-repeat;
+      cursor: pointer;
+      height: $default-grid;
+      margin: 0 auto;
+      opacity: 0.4;
+      width: $double-grid;
+    }
+  }
+
+  #cover {
+    background: $cover-background;
+    box-shadow: 0 0 100px darken($cover-background, 12) inset;
+  }
+
+  #feed {
+    background: $feed-background;
+    box-shadow: 0 0 100px darken($feed-background, 12) inset;
+
+    .feed-example {
+      @extend .l-flex-vertical;
+      flex: 0 0 300px;
+      margin: $default-gap $default-gap $double-gap;
+    }
+
+    .item-wrapper {
+      flex: 0 0 ($default-grid - 2);
+      margin-bottom: $small-gap * 0.5;
+      position: relative;
+      width: 100%;
+    }
+
+    .search-item {
+      @include hidpi-background-image(icon-search, 16px 16px);
+      background-color: $white;
+      background-position: $default-gap center;
+      background-repeat: no-repeat;
+      border: 1px solid $baby-grey;
+      border-radius: $double-gap;
+      flex: 0 0 ($default-grid - 2);
+      margin: 0 auto $small-gap auto;
+      width: 80%;
+    }
+
+    .item {
+      background: $white;
+      border-radius: $double-gap 0 0 $double-gap;
+      box-shadow: 0 1px 1px $baby-grey;
+      height: ($default-grid - 2);
+      width: 100%;
+    }
+
+    .img {
+      background: $baby-grey;
+      border-radius: $round-radius;
+      height: ($default-grid - 2);
+      left: 0;
+      position: absolute;
+      top: 0;
+      width: ($default-grid - 2);
+    }
+  }
+
+  #mobile {
+    background: $mobile-background;
+    box-shadow: 0 0 100px darken($mobile-background, 12) inset;
+
+    .chronicle-devices {
+      @include hidpi-background-image(chronicle-devices, 370px 168px);
+      background-repeat: no-repeat;
+      height: 168px;
+      margin: $default-gap auto;
+      width: 370px;
+
+      @include respond-to(small) {
+        background-size: 310px 141px;
+        height: 141px;
+        width: 310px;
+      }
+    }
+  }
+
+  #promise {
+    background: $promise-background; 
+    box-shadow: 0 0 100px darken($promise-background, 12) inset;
+
+    .mozilla-logo {
+      @include hidpi-background-image(mozilla, 160px 42px);
+      height: 42px;
+      margin: $default-gap auto $quad-gap auto;
+      width: 160px;
+    }
+  }
+
+  #start {
+    box-shadow: 0 0 100px darken($body-background, 12) inset;
+
+    input {
+      font-size: $default-font-size + 2;
+      margin: $double-gap auto;
+      max-width: $default-grid * 10;
+      min-width: $default-grid * 8;
+      padding: $default-gap;
+    }
+  }
+}
+
+/* The following is a lightly modified version of the styles
+*  used by the fullpage plugin.
```

Comment:
@_6a68: 

Yeah, there's a bower installed version of the stylesheet, maybe just add a comment pointing to original?